### ISSUE TITLE
showFailuresOnly is queried in RooibosDist.brs as showOnlyFailures

### ISSUE THREAD

**Issue by ronenlh** [georgejecook/rooibosPreprocessor#10]
`showOnlyFailures` is saved in `configJson`, as well as in `RooibosProcessor` as `showFailuresOnly`,  but in `rooibosDist.brs` it is always queried as `showOnlyFailures`. That might be the reason this feature hasn't worked for me.

**Comment by georgejecook** [comment]
I don't think so, check the runtime config that rooibos generated (it's returned in your rooibos function map), you should observe the setting is honoured.
